perception/sub8_perception/CMakeLists.txt

```diff
@@ -149,61 +83,16 @@ add_dependencies(
 target_link_libraries(
   torpedos_cpp
     sub8_vision_lib
-    # sparsestereo
     ${catkin_LIBRARIES}
     ${Boost_LIBRARIES}
     ${OpenCV_INCLUDE_DIRS}
 )
 
-#add_executable(
-#  perception_server
-#    nodes/perception_server.cpp
-#)
-
-#add_dependencies(
-# perception_server
-#   sub8_msgs_generate_messages_cpp
-#   ${catkin_EXPORTED_TARGETS}
-#)
-
-#target_link_libraries(
-#  perception_server
-#    sub8_vision_lib
-#    ${catkin_LIBRARIES}
-#    ${Boost_LIBRARIES}
-#    ${OpenCV_INCLUDE_DIRS}
-#)
-
-# add_executable(
-#   pcl_buoy
-#     nodes/buoy_node.cpp
-#     ${BUOY_SOURCES}
-# )
-
-# add_dependencies(
-#   pcl_buoy
-#    sub8_msgs_generate_messages_cpp
-#    ${catkin_EXPORTED_TARGETS}
-# )
-
-# target_link_libraries(
-#   pcl_buoy
-#     ${catkin_LIBRARIES}
-#     ${PCL_COMMON_LIBRARIES}
-#     ${PCL_IO_LIBRARIES}
-#     ${PCL_LIBRARIES}
-#     ${Boost_LIBRARIES}
-#     ${OpenCV_INCLUDE_DIRS}
-#     bfd
-#     dl
-# )
```

Comment:
In most cases, there is little chance it will ever be uncommented so it just adds lines to files and makes them harder / more confusing to read. Plus, we have version control if we need to look at an older version of a file.